event-bus/src/sqs.rs

```diff
@@ -24,9 +24,10 @@ pub struct SqsEventBus {
 }
 
 impl SqsEventBus {
-    pub(crate) async fn new() -> Result<Self, anyhow::Error> {
-        let region_provider = RegionProviderChain::default_provider().or_else(Region::new("eu-west-1"));
-        let config = aws_config::from_env().region(region_provider).load().await;
+    pub(crate) async fn new(access_key: String, secret_key: String, region: String) -> Result<Self, anyhow::Error> {
+        let config = SdkConfig::builder()
+            .region(region)
+            .credentials_provider(provider)
```

Comment:
I think this block of code is not needed for fixing the sorting. But I tested this PR and it nicely fixes the sort by severity issue.

Reply:
Ah, this is just a bad commit pulling things I didn't meant to.